### ISSUE TITLE
environment.json is empty on Windows after running yarn gulp build

### ISSUE THREAD

**Issue by lgg** [Jigsaw-Code/outline-client#347]
In Readme.md
> Browser platform development will use fake servers to test successful and unsuccessful connections.

Platform: win7 x64, cygwin:
```
git clone https://github.com/Jigsaw-Code/outline-client
cd outline-client
yarn gulp build --platform=browser --watch
```

Now I want to add any server to check the server's paper layout, but I can't add any fake or real server.

**Comment by alalamav** [comment]
@lgg, you need to copy/paste an access key (ss://...) and add it via the UI. The fake servers should be already displayed when the page loads. 
Are you getting an error?

**Comment by lgg** [comment]
@alalamav just tried again:

Win 7 x64, using cygwin

```
git clone https://github.com/Jigsaw-Code/outline-client
cd outline-client
yarn
yarn gulp build --platform=browser --watch
```

![image](https://user-images.githubusercontent.com/12018713/47469135-07fa9200-d808-11e8-9649-be554430ecb0.png)

after clicking OK

![image](https://user-images.githubusercontent.com/12018713/47469199-4d1ec400-d808-11e8-92b2-ae2a3418f6bb.png)

<details><summary>Outline app looks like</summary><p>

![image](https://user-images.githubusercontent.com/12018713/47469281-c4545800-d808-11e8-8cdc-0116054b3db8.png)

![image](https://user-images.githubusercontent.com/12018713/47469284-c9190c00-d808-11e8-8d2f-51030fee6398.png)
</p></details>

<details><summary>Trying to add real key of real server(built clients for iOS and Win works fine with that key</summary><p>

![image](https://user-images.githubusercontent.com/12018713/47469361-33ca4780-d809-11e8-983b-a18a82fe628c.png)

Pressing Enter and waiting - nothing happens
</p></details>

**Comment by lgg** [comment]
@alalamav error in Dev Console:

```
SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at XMLHttpRequest.xhr.onload (www\app\environment.js:36)
onUnexpectedError @ www\app\main.js:98
www\app\environment.js:36 Uncaught (in promise) SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at XMLHttpRequest.xhr.onload (www\app\environment.js:36)
```

``` javascript
// According to http://caniuse.com/#feat=fetch fetch didn't hit iOS Safari
// until v10.3 released 3/26/17, so use XMLHttpRequest instead.
exports.onceEnvVars = new Promise(function (resolve, reject) {
    var xhr = new XMLHttpRequest();
    xhr.onload = function () {
        try {
            var json = JSON.parse(xhr.responseText); //This line cause error   
            validateEnvVars(json);
            console.debug('Resolving with envVars:', json);
            resolve(json);
        }
        catch (err) {
            reject(err);
        }
    };
    xhr.open('GET', 'environment.json', true);
    xhr.send();
});
```

**Comment by lgg** [comment]
I tried to add gulptask for test:
```
gulp.task('testenv', function() {
  let command = 'scripts/environment_json.sh -p browser > testenv2.json';
  gutil.log(`Running scripts/environment_json.sh -p browser > testenv2.json`);
  let options = {};
  let child = child_process.exec(command, options, function(err, stdout, stderr) {
    gutil.log('Done');
  });
  child.stdout.pipe(process.stdout);
  
  child.stderr.pipe(process.stderr);
  child.stderr.pipe(fs.createWriteStream('./debug.log'));
});

But now i'm stuck with encoding -___-
```

![image](https://user-images.githubusercontent.com/12018713/47470954-ccb09100-d810-11e8-9d87-692ab666c8b6.png)

**Comment by lgg** [comment]
**TL;DR:**
This line 
https://github.com/Jigsaw-Code/outline-client/blob/cec7f69990bf4f0bd5e4b457c56dafe1df5d5ea4/gulpfile.js#L151

Didn't work for Windows, because Windows drops error:
```
'scripts' is not recognized as an internal or external command,
operable program or batch file.
```

 which causes `platforms/browser/www/environment.json` to be empty


Solution:
for Windows change command to `sh ./scripts/environment_json.sh`
via https://stackoverflow.com/questions/26522789/how-to-run-sh-on-windows-command-prompt
